index.bs

```diff
@@ -894,6 +912,65 @@ A {{Point2D}} represents a location in a two dimensional space. The origin of co
   </pre>
 </div>
 
+## Update camera focus distance and {{takePhoto()}} ## {#example4}
+
```

Comment:
I'll leave things as is for now, sounds like there's still some discussion on the content of the examples which may cause them to be revised in the near future anyway.